miso-web/src/it/resources/db/scripts/schema_translator.groovy

```diff
@@ -18,6 +18,7 @@ println('Translating schema files from ' + productionSchemaDir.getAbsolutePath()
 
 final String productionScriptPattern = '^(V\\d{4}_.*|afterMigrate|beforeMigrate)\\.sql$'
 final String testSchemaDir = basedir + '/target/test-classes/db/migration/'
+final String testStoredProceduresFile = basedir + '/src/it/resources/db/migration/afterAfterMigrate.sql'
```

Comment:
Can't you just put this file in `migration_afterMigrate` and name it `99_test_procedures.sql` or something similar to have it run after all the other afterMigrate scripts, which are all numbered `05`?

Reply:
Unfortunately, we can't put it into `migration_afterMigrate` until we convert the unit tests to MySQL. Straight up moving it there causes the tests to fail.

Reply:
`-- StartNoTest`?

Reply:
:bulb: ...it works!